File: tmp-mei/fgen4255027576711118260/base/ControlFlow_0.java
package base;
public interface ControlFlow_0{
ControlFlow_0 $self = new ControlFlow_0Impl();
base.ControlFlow_1 continue$imm();

base.ControlFlow_1 return$imm(Object returnValue_m$);

base.ControlFlow_1 breakWith$imm();

base.ControlFlow_1 continueWith$imm();

base.ControlFlow_1 break$imm();
static base.ControlFlow_1 continue$imm$fun(base.ControlFlow_0 $this) {
  return ((base.ControlFlow_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.ControlFlow_1) null;
}})
);
}

static base.ControlFlow_1 break$imm$fun(base.ControlFlow_0 $this) {
  return ((base.ControlFlow_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.ControlFlow_1) null;
}})
);
}

static base.ControlFlow_1 continueWith$imm$fun(base.ControlFlow_0 $this) {
  return ((base.ControlFlow_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.ControlFlow_1) null;
}})
);
}

static base.ControlFlow_1 breakWith$imm$fun(base.ControlFlow_0 $this) {
  return ((base.ControlFlow_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.ControlFlow_1) null;
}})
);
}

static base.ControlFlow_1 return$imm$fun(Object returnValue_m$, base.ControlFlow_0 $this) {
  return ((base.ControlFlow_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.ControlFlow_1) null;
}})
);
}
}

File: tmp-mei/fgenjs5030319945734616766/base/ControlFlow_0.js
import * as rt from "../rt/main.js";
export class base$$ControlFlow_0 {
  static $self = new base$$ControlFlow_0();

  continue$imm() {
  return base$$ControlFlow_0.continue$imm$fun(this);
}


return$imm(returnValue_m$) {
  return base$$ControlFlow_0.return$imm$fun(returnValue_m$, this);
}


breakWith$imm() {
  return base$$ControlFlow_0.breakWith$imm$fun(this);
}


continueWith$imm() {
  return base$$ControlFlow_0.continueWith$imm$fun(this);
}


break$imm() {
  return base$$ControlFlow_0.break$imm$fun(this);
}

  static continue$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static break$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static continueWith$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static breakWith$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static return$imm$fun(returnValue_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}
